code/datums/station_traits/neutral_traits.dm

```diff
@@ -127,3 +127,57 @@
 
 	var/new_colored_assistant_type = pick(subtypesof(/datum/colored_assistant) - get_configured_colored_assistant_type())
 	GLOB.colored_assistant = new new_colored_assistant_type
+
+/datum/station_trait/cargorilla
+	name = "Cargo Gorilla"
+	trait_type = STATION_TRAIT_NEUTRAL
+	weight = 1
+	show_in_report = FALSE // Selective attention test. Did you spot the gorilla?
+
+	/// The gorilla we created, we only hold this ref until the round starts.
+	var/mob/living/simple_animal/hostile/gorilla/cargo_domestic/cargorilla
+
+/datum/station_trait/cargorilla/New()
+	. = ..()
+	RegisterSignal(SSatoms, COMSIG_SUBSYSTEM_POST_INITIALIZE, .proc/replace_cargo)
+
+/// Replace some cargo equipment and 'personnel' with a gorilla.
+/datum/station_trait/cargorilla/proc/replace_cargo(datum/source)
+	SIGNAL_HANDLER
+
+	var/mob/living/simple_animal/sloth/cargo_sloth = locate(/mob/living/simple_animal/sloth/citrus) || locate(/mob/living/simple_animal/sloth/paperwork)
+	if(!cargo_sloth)
+		return
+
+	cargorilla = new(cargo_sloth.loc)
+	cargorilla.name = cargo_sloth.name
+	cargorilla.being_polled_for = TRUE // We do a poll on roundstart
```

Comment:
if this poll fails, let people double-click the cargorilla to take them over as a ghost

Reply:
they can, it's in attack_ghost